paper.tex

```diff
@@ -99,11 +99,6 @@
 We provide thoroughly tested implementations of Demes parsers in Python
 and C, and showcase initial support in a number of simulators and inference
 methods.
-
```

Comment:
I think we're splitting hairs here - we showcase them by mentioning them in the table and we also definitely showcase moments.

Reply:
I read this as "we're going to show you how to do some parameter inference using Demes". But :shrug:

Reply:
I see - maybe you could tweak the language a bit there then?